File: data/models/src/main/kotlin/red/razvan/restcountries/data/models/CountryListItem.kt
// Copyright 2025 Răzvan Roșu
// SPDX-License-Identifier: Apache-2.0

package red.razvan.restcountries.data.models

data class CountryListItem(
  val id: CountryId,
  val officialName: String,
  val emojiFlag: String,
)

File: ios-app/rest-countries/Sources/Models/CountryListItem.swift
//
//  CountryListItem.swift
//  rest-countries
//
//  Created by Răzvan Roşu on 13/08/25.
//

public struct CountryListItem: Identifiable, Hashable {
  public let id: CountryId
  public let officialName: String
  public let emojiFlag: String
}
